src/icons/SealEye.tsx

```diff
@@ -1,9 +1,64 @@
-import { stroke } from 'classnames/tailwind'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
 
-const strokeSecondaryDimmed = stroke('stroke-secondary-dimmed')
-const strokeTertiaryLine = stroke('stroke-tertiary')
+enum SealEyeColor {
+  accent = 'stroke-accent',
+  secondary = 'stroke-secondary-dimmed',
+  tertiary = 'stroke-tertiary',
+}
+
+const commonStrokeWidth = strokeWidth('stroke-1.5')
+
+export default function ({ reversed }: { reversed?: boolean }) {
+  const strokeClass = (color: SealEyeColor) =>
+    classnames(commonStrokeWidth, stroke(color))
+
+  if (reversed)
+    return (
+      <svg
+        width="55"
+        height="46"
+        viewBox="0 0 55 46"
+        fill="none"
```

Comment:
This is set by default, let's remove it

Reply:
Hmm... I've removed it before. Did it again